include/engine/descriptors/config.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
I think `osrm-backend` consistently uses define-include guards instead of pragmas